### ISSUE TITLE
Discussion: Should we put LXQt About in the first level of app menu?

### ISSUE THREAD

**Issue by PCMan** [lxde/lxqt#740]
In commit b10356d of lxqt-common, a menu entry for "LXQt About" is created in the first level of the app menu.
https://github.com/lxde/lxqt-common/commit/b10356d26e784c92a2e49cf207f889315de77b51
Is it really a good idea to put LXQt About in this place?
While this indeed helps the branding of our project, its benefit in usability is not that obvious.
Given the limited space of menu, especially for devices with small screens (netbooks), the precious space of first-level menu is reserved for most frequently used apps in most cases.
LXQt About is not a program that an ordinary user may use everyday.
Most of the time, the user might only use it once.
IMHO a rarely used program should not occupy a menu entry in the first-level menu.
What do you think?

**Comment by agaida** [comment]
imho we should leave it there for now - i don't know how you like the idea - but i would suggest to improve our start menu a little bit in the direction of kde kicker, add a picture later - this would give us room in the menu and improve usability.

**Comment by jleclanche** [comment]
I'm highly in favour of getting an advanced application menu but we will probably need to keep our current menu plugin as well for those that prefer the simple version.

**Comment by agaida** [comment]
ok - a switch to turn a panel on or off should be not a big problem i hope

**Comment by antis81** [comment]
It would be ok to place `lxqt-about` as a top-level entry in the menu, if it would integrate better with system profiler of the distro. From a user's perspective, what I want to see is a summary of my system (especially, becaused I am using a customized distro).

My question is: Does it make sense to integrate the `lxqt-about` dialog better with the "system profiler" - probably add a button: `Show detailed system information`)? 

Lubuntu's system profiler currently provides only rough info about LXQt (it's name only):

![bildschirmfoto von 2015-08-21 10 11 52](https://cloud.githubusercontent.com/assets/440517/9404272/6524078e-47ee-11e5-8d5b-307039118db9.png)

Proposal for `lxqt-about` (changes in orange):

![lxqt-about-proposed](https://cloud.githubusercontent.com/assets/440517/9404654/a34fe14c-47f1-11e5-8666-ac446486953e.png)

**Comment by jleclanche** [comment]
No, we're not doing that, although the about dialog could list git commit ids if possible...

**Comment by antis81** [comment]
Yes, this is totally possible and very helpful for tracking down user's issues - even for people without deeper knowledge about LXQt.
